enclave_ops/deployment/dkeycache/Enclave/enclave_la.cpp

```diff
@@ -465,22 +446,33 @@ extern "C" ATTESTATION_STATUS enclave_la_generate_response(secure_message_t* req
                 sizeof(temp_resp_message->message_aes_gcm_data.reserved), plaintext, plaintext_length,
                 &(temp_resp_message->message_aes_gcm_data.payload_tag));
 
-    if(SGX_SUCCESS != status)
-    {
-        SAFE_FREE(resp_data);
-        SAFE_FREE(decrypted_data);
-        SAFE_FREE(temp_resp_message);
-        return status;
-    }
+    if (SGX_SUCCESS != status)
+        goto clean;
 
     memset(resp_message, 0, sizeof(secure_message_t)+ resp_data_length);
     memcpy(resp_message, temp_resp_message, sizeof(secure_message_t)+ resp_data_length);
```

Comment:
does it need to change to meeset_s?

Reply:
No.  it will not be compiler optimized because resp_message is used @453.